### ISSUE TITLE
cnf-flip json->yaml conversion creates invalid linewrap ?

### ISSUE THREAD

**Issue by pgnd** [awslabs/aws-cfn-template-flip#100]
for input,

test.json:
```
{
      "app.example.com" : {
         "(?#Configuration)^/(.*?\\.(fcgi|psgi)/)?(manager\\.html|confs/|$)" : "inGroup(\"admingroup\")",
         "(?#Notifications)/(.*?\\.(fcgi|psgi)/)?notifications" : "inGroup(\"admingroup\") or $uid eq \"myuser\"",
         "(?#Sessions)/(.*?\\.(fcgi|psgi)/)?sessions" : "inGroup(\"admingroup\") or $uid eq \"myuser\"",
         "default" : "inGroup(\"admingroup\") or $uid eq \"myuser\""
      }
}
```

which, checked @ https://jsonlint.com/?json=, returns 'Valid JSON'

exec of
```
cfn-flip test.json test.yaml
```

returns output,

test.yaml:
```
app.example.com:
  (?#Configuration)^/(.*?\.(fcgi|psgi)/)?(manager\.html|confs/|$): inGroup("admingroup")
  (?#Notifications)/(.*?\.(fcgi|psgi)/)?notifications: inGroup("admingroup") or $uid
    eq "myuser"
  (?#Sessions)/(.*?\.(fcgi|psgi)/)?sessions: inGroup("admingroup") or $uid eq "myuser"
  default: inGroup("admingroup") or $uid eq "myuser"
```

note the NEWLINE
```
    eq "myuser"
```

which is invalid wrapped/indented yaml

is this a source or config problem, or a bug?

**Comment by koiker** [comment]
The code is using the default max width of PyYaml and the output is creating a newline.
Besides the newline, the yaml output is valid and you can test the conversion again to json that will result in a valid json.

> sample.yaml file:
```
app.example.com:
  (?#Configuration)^/(.*?\.(fcgi|psgi)/)?(manager\.html|confs/|$): inGroup("admingroup")
  (?#Notifications)/(.*?\.(fcgi|psgi)/)?notifications: inGroup("admingroup") or $uid
    eq "myuser"
  (?#Sessions)/(.*?\.(fcgi|psgi)/)?sessions: inGroup("admingroup") or $uid eq "myuser"
  default: inGroup("admingroup") or $uid eq "myuser"%
```

```
cfn-flip -i yaml sample.yaml
{
    "app.example.com": {
        "(?#Configuration)^/(.*?\\.(fcgi|psgi)/)?(manager\\.html|confs/|$)": "inGroup(\"admingroup\")",
        "(?#Notifications)/(.*?\\.(fcgi|psgi)/)?notifications": "inGroup(\"admingroup\") or $uid eq \"myuser\"",
        "(?#Sessions)/(.*?\\.(fcgi|psgi)/)?sessions": "inGroup(\"admingroup\") or $uid eq \"myuser\"",
        "default": "inGroup(\"admingroup\") or $uid eq \"myuser\""
    }
}%
```


But I will change the default width to 120 and add an environment variable or parameter to allow users change this behavior.

**Comment by ctd** [comment]
@koiker I've assigned this issue to you since I assume you're working on a patch.

Feel free to un-assign if that's not the case.

**Comment by koiker** [comment]
OK, let me get some spare time to apply the change.

**Comment by koiker** [comment]
Fixed in #102
